README.md

```diff
@@ -1,17 +1,29 @@
 # Collect and store RouteViews data, from a Cloud Service
 
+A collaboration between Route Views and Google to expand Route Views data offerings 
```

Comment:
as with the mlab comment/info ... I think i destroyed this commit in my git actions :(
these all look ok to me though :)